src/test/java/com/example/delivery_system/service/CarriageServiceTest.java

```diff
@@ -0,0 +1,84 @@
+package com.example.delivery_system.service;
+
+import com.example.delivery_system.entity.Carriage;
+import com.example.delivery_system.entity.Outlet;
+import com.example.delivery_system.repository.CarriageRepository;
+import com.example.delivery_system.repository.OrderRepository;
+import com.example.delivery_system.repository.RouteRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class CarriageServiceTest {
+    @Mock
+    CarriageRepository carriageRepository;
+    @Mock
+    RouteRepository routeRepository;
+    @Mock
+    OrderRepository orderRepository;
+
+    CarriageService carriageService;
+
+    @BeforeEach
+    void setUp() {
+        carriageService = new CarriageService(carriageRepository,routeRepository,orderRepository);
+    }
+
+    @Test
+    void findCarriageById() {
+        when(carriageRepository.findById(1L)).thenReturn(Optional.of(new Carriage()));
+        Carriage carriage = carriageService.findCarriageById(1L);
+        assertNotNull(carriage);
```

Comment:
It would be nice to check here that the result is not only not null but check value of all fields as well